gui/src/components/VersionInfo.svelte

```diff
@@ -8,22 +8,32 @@
 
   let installedVersion: string | null = null;
   let latestVersion: string | null = null;
+  let isManaged: boolean = false;
   let upgrading = false;
 
   const doUpgrade = async () => {
+    if (isManaged) {
+      alert(
```

Comment:
Do we have nothing better than `alert`?

Reply:
Not right now. @jwmza

Reply:
Modal system coming imminently, I'm going to wrap up some in-flight things first though.